test/views/pages/post/static_post_page_test.dart

```diff
@@ -86,17 +86,47 @@ void main() {
       final postUserBar = find.byKey(CompletePostWidget.postUserBarKey);
       expect(postUserBar, findsOneWidget);
 
+      //Check that the owner display name is displayed
       final postUserBarDisplayNameTextWidget = tester.widget(
         find.descendant(
           of: postUserBar,
-          matching: find.byKey(UserBarWidget.displayNameTextKey),
+          matching: find.byKey(PostHeaderWidget.displayNameTextKey),
         ),
       );
+
       expect(
         postUserBarDisplayNameTextWidget is Text &&
             postUserBarDisplayNameTextWidget.data == post.ownerDisplayName,
         true,
       );
+
+      //Check that the publication time is displayed
+      final postUserBarTimestampTextWidget = tester.widget(
+        find.descendant(
+          of: postUserBar,
+          matching: find.byKey(PostHeaderWidget.timestampTextKey),
+        ),
+      );
+
+      expect(
+        postUserBarTimestampTextWidget is Text &&
+            postUserBarTimestampTextWidget.data ==
+                timestampToTimeAgo(post.publicationTime),
+        true,
+      );
+
+      // Check Tooltip message
+      final tooltip = find.byType(Tooltip);
+      expect(tooltip, findsOneWidget);
+
+      final tooltipWidget = tester.widget(tooltip);
+      expect(tooltipWidget is Tooltip, true);
+
+      final tooltipMessage = (tooltipWidget as Tooltip).message;
+      expect(
+        tooltipMessage,
+        timestampToDate(post.publicationTime),
+      );
```

Comment:
I think that we shouldn't use directly `timestampToTimeAgo` and `timestampToDate` from `lib/views/home_content/feed/post_card/post_header_widget.dart` but rather directly give the expected string. 
That way, if we change `timestampToTimeAgo` or `timestampToDate` and make a mistake we want to catch it. 

It also imply that changing `timestampToTimeAgo` or `timestampToDate` imply careful and conscious implications and not direct test pass as the function is shared. 😄

Reply:
You mean harcode the expected result ? Or "duplicate" `timestampToTimeAgo`?
It would be difficult to hard code it in the case of `timeago`, since it keeps changing.

Reply:
Yes you are right. I think that an idea would be to fix the time of the emulator, but it may be not trivial and would imply refactor of other tests. I think that we can duplicate the method for now and add to the backlog to add tests that fixes the time to really put the expected result as string 😄

Reply:
see https://github.com/ProximaEPFL/proxima/pull/196#issuecomment-2090557662 for my take on testing

Reply:
I will go with Guillaume suggestion for now and leave a TODO.
The creation of a service is a very good idea, and after discussion with our scrum master @JoachimFavre, we decided to add it to the backlog.

Reply:
Implemented Guillaume's suggestion in 38d616f1df08f7930974dfdf464ef1ca5794c0e6
Created issue #208 in backlog